LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Daniel Dudt, Rory Conlin, Dario Panici, Egemen Kolemen
+Copyright (c) 2020 Daniel Dudt, Rory Conlin, Dario Panici, Kaya Unalmis, Yigit Elmacioglu, Rahul Gaur, Patrick Kim, Egemen Kolemen
```

Comment:
does the year need updating if we are changing this? or do we keep this the same as this was the original license? I have no idea how license stuff works

Reply:
https://stackoverflow.com/a/20911485

Reply:
If you expect to live to 150 years and would like to keep the copyright for a few more years you could update it